etc/syntax/reference-files/suite_jinja2.rc

```diff
@@ -0,0 +1,70 @@
+#!Jinja2
+# ^ SYNTAX:3.1
+
+# SYNTAX numbering matches the table in https://github.com/cylc/cylc/issues/2752
+
+# SYNTAX: 3.2
+# templating code should be displayed diffently to cylc code
+{# no comment #}  # should display as a comment
+{% set foo = "bar" %}
+{% set foo = '%s' % 'bar' %}
+{% set foo = {"1": 2, "3", []} %}
+{% set foo = foo.bar(1, 2, 3) %}
+{{ foo_bar }}
+{{ foo.bar(1, 2, 3) }}
+{{ foo | bar(1, 2, 3) }}
+{% include 'foo' %}
+{% include ['foo', 'bar'] %}
+{% import 'foo' as foo %}
+{% from 'foo' import foo as a, b %}
+
+# SYNTAX 3.3
+# multi-line templating code should be consistent with single-line code
+{#
+    no comment
+#}  # all three lines should display as a single comment
+{# a
+   b
+   c #}  # all three lines should display as a single comment
+{%
+    set foo = {"1": 2, "3", []}
+%}
+{% set foo = {
+    "1": 2,
+    "3", []} %}
+
+
+
+# SYNTAX 3./1.1
+# keys, values and sections outside of the templating code should display
+# as normal
+a = {{ b }}
+{{ a }} = b
+a = {{ b + 1 }}
+a = {{ b_b | c.d('1', "2", 3) }}
+[[{{ a }}]]
+    d = e  # check highlighting of this line un-affected by section
+[[{{ a_a | c.d('1', "2", 3) }}]]
+    d = e  # check highlighting of this line un-affected by section
+[[a{{ b }}c]]
+    d = e  # check highlighting of this line un-affected by section
+[[{{ "%(foo)s" % {"foo": "abc"}}}]]
```

Comment:
```suggestion
[[{{ "%(foo)s" % {"foo": "abc"}}]]
```

As picked up by my emacs lexer! :grin:

Reply:
The original is actually right!

```
[[{{ "%(foo)s" % {"foo": "abc"}} }]]
```

Reply:
Oops, it is indeed right. Sorry for my mistake :no_mouth:.

> As picked up by my emacs lexer!

It brings to mind that old saying **a bad workman blames his [/her] tools**. I really can't blame the emacs lexer for pointing out something that is actually correct, since I wrote it (or at least the part that should pick that up properly)!